internal/cli/init.go

```diff
@@ -14,10 +14,18 @@ import (
 
 func initCmd() *cobra.Command {
 	initCmd := &cobra.Command{
-		Use:     "init",
-		Short:   fmt.Sprintf("Initialize a new %s repo", program.Name),
-		Long:    fmt.Sprintf("Initialize a new %s repo", program.Name),
-		Args:    cobra.ExactArgs(1),
+		Use:   "init",
+		Short: fmt.Sprintf("Initialize a new %s repo", program.Name),
+		Long:  fmt.Sprintf("Initialize a new %s repo", program.Name),
+		// Args:  cobra.ExactArgs(1),
+		Args: func(cmd *cobra.Command, args []string) error {
+			fn := cobra.ExactArgs(1)
+			err := fn(cmd, args)
+			if err != nil {
+				return errors.New("you must specify a path to a repo you want cavorite to track")
+			}
+			return nil
+		},
```

Comment:
I assume the default error here is too generic and not super helpful.

Reply:
yep exactly. default error was not helpful